### DISCUSSION TITLE
food/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/food#3] [General]
# food/todo/

https://shanenull.com/food/todo/

**Comment by shane0** [comment]
# recipes

- dessert https://youtube.com/shorts/d5ueqB5QseU?si=QZuprkHOFWp62RL3

**Reply by shane0** [reply]
# Japanese

https://www.kamaboko.com/en/castella-style-tamagoyaki-cooked-fluffy-eggs/

**Reply by shane0** [reply]
https://www.snowpeak.com/blogs/news/tramezzino-recipe-roundup

**Comment by shane0** [comment]
sunchokes jerusalem artichoke

**Comment by alfahu27** [comment]
Hi i am Alfa, i think your readers will like our article about [healthy food](https://www.lemonilo.com/blog). maybe it could be an additional reference on your site?

**Comment by shane0** [comment]
Goldenrod
https://youtu.be/hsbKHGuMZ64?si=LWbqDyFoMArAzWcc

**Comment by shane0** [comment]
Document useful plants I’ve found or can find 
Locally

Ghost pipe

https://youtu.be/peBhUI-XXGQ?si=U1vUfixNAM4SXiLi

**Comment by shane0** [comment]
# foraging page

- burdock

<iframe width="950" height="534" src="https://www.youtube.com/embed/oSyRZTG-wPA" title="BURDOCK ROOT - Uses, health &amp; medicinal benefits &amp; identification" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

**Reply by shane0** [reply]
Paste my fishing books here
Burdock
Lambs quarters
https://www.instagram.com/reel/DIXoThxS2GZ/?igsh=NTc4MTIwNjQ2YQ==

**Comment by shane0** [comment]
![IMG_3685](https://github.com/shane0/food/assets/1454458/929a16f8-3ad2-4fca-ba7e-9d31cab56df6)

**Comment by shane0** [comment]
![f](https://private-user-images.githubusercontent.com/1454458/286499932-9948ff50-008f-4257-a1f4-1bdd3ea14952.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEyNDAyNzEsIm5iZiI6MTcwMTIzOTk3MSwicGF0aCI6Ii8xNDU0NDU4LzI4NjQ5OTkzMi05OTQ4ZmY1MC0wMDhmLTQyNTctYTFmNC0xYmRkM2VhMTQ5NTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTEyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzExMjlUMDYzOTMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjY2YWE0MzhkOTcyMWI1ODg4Nzg2MjYwNWM4YzM5ZjM0NDI1ZjhkODM0MWM3MmMyNTNiYjZkNjU3NzNkYzdjYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.KZdR5kugMZ-IzVuU3nRWli0RPyrRBmZVyMigUq28WuU)

**Reply by shane0** [reply]
Seafood
 House foods 
Aqua treasure

**Reply by shane0** [reply]
https://www.facebook.com/people/Aqua-Treasure/100078987978278/

**Comment by shane0** [comment]
![IMG_3709](https://github.com/shane0/food/assets/1454458/fa8dee05-fe88-4b2f-824f-e24139fc3f13)

**Comment by shane0** [comment]
https://www.masteringdiabetes.org/pea-and-fava-bean-mint-toast/

**Comment by shane0** [comment]
All day energy bowl
https://youtube.com/shorts/jsXF-4UsW8U?si=JBy5TsKy998VYZul

**Comment by shane0** [comment]
broccoli 425 degrees 18 minutes

**Comment by shane0** [comment]
sweet potatoes cubes 425 22 minutes

**Comment by shane0** [comment]
Steps
https://youtube.com/shorts/f_PopiMhjoU?si=4QC9dfG8jnOkiXs3

**Comment by shane0** [comment]
dry chickpeas to a large mixing bowl, add, and the salt. Give it a good mix. Add your flavor mix-ins. Spread the chickpeas on a lined baking tray. Bake the chickpeas for 45 minutes at 200C/400F, stirring the chickpeas every 15 minutes. 


https://thebigmansworld.com/healthy-oil-free-roasted-chickpeas-vegan-gluten-free-grain-free/

**Comment by shane0** [comment]
https://www.masteringdiabetes.org/crunchy-food-cravings/

**Comment by shane0** [comment]
# hummus no oil


```
3 cups cooked chickpeas (drained and rinsed)
6 Tbsp lemon juice
6 tsp garlic, minced
3-4 tsp chili powder
1 tsp cumin
2½ tsp Dijon mustard
Salt/pepper to taste (optional)
Water as needed

```

**Comment by shane0** [comment]
fries russet

spuds in a 425F, set a timer for 35 minutes, and relax. No need to flip, watch, or broil

**Comment by shane0** [comment]
chickpea patties

https://youtu.be/eoF6vjGQQzQ

**Comment by shane0** [comment]
chickpea meatballs https://youtu.be/xtdJv4I08MA

**Comment by shane0** [comment]
https://www.ctworld.org.tw/english-96/html/06_Recipes.html

**Comment by shane0** [comment]
lentil tofu https://youtu.be/4aqx69E9T4A?si=yp5NwHZO-jw4LGW3

**Comment by shane0** [comment]
**Here are simple instructions for cooking quinoa, accompanied by images for visual clarity:**

**Ingredients:**

- 1 cup quinoa
- 1 3/4 cups water or broth
- Pinch of salt (optional)

**Steps:**

1. **Rinse the quinoa:**
   - Place the quinoa in a fine-mesh sieve and rinse it thoroughly under cold running water for about 30 seconds. This helps remove any saponins, which can have a bitter taste.
   [Image of Rinsing quinoa in a sieve under running water]

2. **Combine quinoa and water:**
   - Place the rinsed quinoa in a medium saucepan.
   - Add the water or broth and salt (if using).
   [Image of Quinoa and water in a saucepan]

3. **Bring to a boil:**
   - Cover the saucepan with a lid and bring the mixture to a boil over medium-high heat.

4. **Reduce heat and simmer:**
   - Once boiling, reduce the heat to low and simmer, uncovered, for 15-20 minutes, or until the quinoa is tender and the liquid has been absorbed.
   [Image of Simmering quinoa in a saucepan]

5. **Fluff and steam:**
   - Remove the saucepan from heat.
   - Fluff the quinoa with a fork to separate the grains.
   - Cover the saucepan again and let it steam for 5-10 minutes. This helps the quinoa become even more fluffy and light.
   [Image of Fluffing quinoa with a fork]

6. **Serve and enjoy:**
   - Your perfectly cooked quinoa is ready to serve! Enjoy it as a side dish, in salads, bowls, or as a base for main courses.
   [Image of Cooked quinoa in a bowl]

**Additional tips:**

- **Flavor variations:** To enhance the flavor, try toasting the quinoa in a dry saucepan for a few minutes before adding the water. You can also cook quinoa in seasoned broth or add herbs and spices during cooking.
- **Storing quinoa:** Cooked quinoa can be stored in an airtight container in the refrigerator for up to 4 days.
- **Freezing quinoa:** Cooked quinoa can also be frozen for up to 3 months.

**Comment by shane0** [comment]
- [cabot cheese](https://cabotcreamery.com/products/cheese-extra-sharp-bar-deli)
- [seriously sharp](https://cabotcreamery.com/products/copy-of-sharp-cheddar-cheese-snacks-50-ct)
- [Rigani Loukaniko](https://www.olympiaprovisions.com/collections/deli-meats/products/rigani-loukaniko)
- [salami capri](https://www.olympiaprovisions.com/collections/deli-meats/products/salami-capri)

**Comment by shane0** [comment]
https://www.olympiaprovisions.com/blogs/blog-recipes/build-the-perfect-charcuterie-board

**Comment by shane0** [comment]
Lemon blast
https://youtu.be/QZl3CtFoDc4?si=DmmXhYeleCRTkOPF

**Comment by shane0** [comment]
1 Peeled Lemon
2 Apples
1/2 Habanero (full pepper for more advanced)
3 inches Fresh Ginger
3 inches Fresh Turmeric Root (when available)
1 Cucumber
1 Bunch Celery
1 Bunch Cilantro
1 Bunch Parsley (regular or Italian)
1 Handful Fresh Mint
1. Juice all the ingredients
2. Cut final juice amount with 1/3 water.
3. Drink

https://puralive.com/how-to-make-the-lemon-ginger-blast/

**Comment by shane0** [comment]
At Portland store got

 Tea pot nesting mugs belt bag hat
https://www.snowpeak.com/products/sayou-tea-pot

**Comment by shane0** [comment]
https://marketspice.com/products/marketspice-cinnamon-orange-black-tea-16oz-package

**Comment by shane0** [comment]
https://www.theinfatuation.com/seattle/reviews/alebrijes-kitchen?ifsb=yes

**Comment by shane0** [comment]
q62 effortless book on vegetarian

**Comment by shane0** [comment]
carrots
https://youtu.be/ll8pCBWED7A?si=j8enmKoIT-G6irjK

**Comment by shane0** [comment]
# foraging

- add a page w/ notes from my ebooks
- https://youtu.be/WIRemyGT_xM

**Comment by shane0** [comment]
kuzwa

fisherking shrimp tempura
chowder
mac
black cod

**Comment by shane0** [comment]
Thyme 
https://youtu.be/aKw6aeDb8r8?si=gRASVd3qGvRPdkMb

**Comment by shane0** [comment]
https://www.clydesvegetableplantingchart.com/
https://www.sunoven.com/products/sun-oven-doug-and-stacy-edition
https://www.masontops.com/collections/off-grid-with-masontops?utm_source=OFFGRID&utm_campaign=OFF-GRID-YOUTUBE-Mar2017&utm_medium=OFF+GRID+YouTube
https://www.homesteadinglifeconference.com/

**Comment by shane0** [comment]
https://www.olympiaprovisions.com/blogs/blog-recipes/olympia-provisions-laser-potatoes

**Comment by shane0** [comment]
# karaage

https://www.youtube.com/watch?v=rxetIbhWigo&t=30s

https://www.snowpeak.com/blogs/explore/takibi-chicken-karaage

**Reply by shane0** [reply]
2 chicken thighs cut into bite-size pieces  
1 cup shio koji 
1 cup usukuchi shoyu 
1/2t. Minced garlic 
1/2t. Grated ginger 
1 cup rice flour 
1 cup potato starch

**Comment by shane0** [comment]
# carrot top pesto

https://theproducepack.com/carrot-top-pesto/#mv-creation-30-jtr

ngredients
Pesto
Tops from 2 bunches of carrots, approximately 2 cups roughly chopped
½ cup walnut halves
½ cup grated parmesan
2 tbsp fresh squeezed orange juice
1 clove garlic
½ tsp kosher salt
¼ tsp crushed red pepper flakes
¼ cup extra virgin olive oil
To serve
8oz rigatoni pasta, boiled and drained
2 tbsp carrot top pesto
Freshly grated parmesan
Instructions
Pesto
Add carrot tops, walnuts, parmesan, orange juice, garlic, salt and crushed red pepper flakes to a food processor and pulse until finely chopped.
Turn the food processor on, and while it is running, slowly drizzle in the olive oil until a paste forms.

Serving Suggestion
Stir carrot top pesto into cooked pasta immediately after the pasta is drained. The heat from the pasta will warm the pesto sauce.
Top plates with freshly grated parmesan

**Comment by shane0** [comment]
# smokemeister

https://uk.snowpeak.com/blogs/journal/smokemeister-recipe

https://www.snowpeak.com/blogs/explore/smokemeister-recipes

**Comment by shane0** [comment]
# bread

- ezekiel at the moment has 0g sugar + no seed oil
- https://youtu.be/q6-eJKZC8sU

**Comment by shane0** [comment]
# plastic oils

- https://youtu.be/Ra_tCL5-4c0

**Reply by shane0** [reply]
https://youtu.be/9TtamoH-zzA

**Comment by shane0** [comment]
# toxic food

- greeds ruining food 
- https://www.instagram.com/reel/DDZmA7utOuC/?igsh=NHBlMXNpZjMwbTF4

**Reply by shane0** [reply]
Shrinkflation

**Comment by shane0** [comment]
Broccoli air fry salt pepper coat with bacon grease 425 18m

**Comment by shane0** [comment]
Indian style pickled mixed vegetables garlic etc tomato spicy base

**Comment by shane0** [comment]
# Mexican

Chorizo plate

Esquite cast iron
https://youtu.be/ypwwRYdav2c?si=7YDrFBsy4QTdkmZQ

**Comment by shane0** [comment]
# ferment & pickle

cucumber onion from bbq in san diego

carrot cabbage https://youtu.be/Eke87NVsKi0

find the indian recipe i had in monterey at the indian buffet

base was tomato similar to pizza sauce had garlic tomato etc spicy pizza style

find the recipe from the al pastor pop up carrot lime & something else mild ferment

**Reply by shane0** [reply]
https://www.indianhealthyrecipes.com/andhra-tomato-pickle-nilava-tomato-pachadi/

**Reply by shane0** [reply]
https://platinitwithwendy.com/blog/authentic-goan-prawn-balchao/

**Reply by shane0** [reply]
# pickled onion

https://www.youtube.com/watch?v=id1pgfUjzUc

**Reply by shane0** [reply]
ginger beers https://youtu.be/kDgkLzjPZJ8

**Comment by shane0** [comment]
# archar

There are several types of **achars (pickles)** that incorporate **garlic** and have a flavor profile similar to **pizza sauce**—tangy, slightly sweet, and savory. Here are some popular varieties you might enjoy:

### 1. **Garlic Chili Achar (Lahsun Mirch Ka Achar)**  
   - Made with **garlic, red chilies, tomatoes, vinegar, and spices**.  
   - The tangy, slightly sweet, and spicy taste can resemble pizza sauce.  

### 2. **Tomato Garlic Achar**  
   - Contains **tomatoes, garlic, mustard oil, and spices** like cumin, fenugreek, and red chili.  
   - The tomato base gives it a **pizza sauce-like** flavor with a spicy kick.  

### 3. **Sweet & Sour Garlic Pickle**  
   - Combines **garlic, jaggery/sugar, tamarind, and spices** for a balanced sweet-tangy taste.  
   - Some versions include tomatoes, enhancing the pizza sauce resemblance.  

### 4. **Italian-Style Garlic & Herb Pickle (Fusion Achar)**  
   - A modern twist with **garlic, sun-dried tomatoes, oregano, basil, and olive oil**.  
   - Almost like a **spicy, pickled version of pizza sauce**.  

### 5. **Hyderabadi Tomato Achar**  
   - Features **tomatoes, garlic, mustard, and tamarind** for a bold, tangy-sweet flavor.  
   - The garlic and tomato combo makes it taste like a spicier pizza sauce.  

### **Where to Find Them?**  
- Indian grocery stores (look for **Mother’s Recipe, Priya, or homemade brands**).  
- Some specialty pickle makers sell **Italian-inspired garlic-tomato achar**.  

Would you like a recipe to make your own? 😊

**Reply by shane0** [reply]
# **fusion samosa recipe** 

garlic-herb pickle filling, parmesan, pineapple, and store-bought samosa wraps** for an Italian-Indian twist!  

### **Italian-Indian Fusion Samosa Recipe**  
*(Makes 8-10 samosas)*  

#### **Ingredients**  
**For the Filling:**  
- **1 cup Italian-Style Garlic & Herb Pickle** (from the previous recipe, finely chopped)  
- **½ cup cooked mashed potato** (for binding)  
- **¼ cup grated parmesan cheese** (+ extra for topping)  
- **¼ cup finely diced pineapple** (fresh or canned, drained)  
- **1 tbsp chopped fresh cilantro or basil** (optional)  
- **1 tsp chaat masala** (or Italian seasoning for less spice)  
- **Salt & pepper** to taste  

**Other:**  
- **Store-bought samosa wraps** (or spring roll sheets)  
- **Oil** for frying (or bake for a healthier option)  

#### **Instructions**  
1. **Prepare Filling**  
   - In a bowl, mix **garlic-herb pickle, mashed potato, parmesan, pineapple, cilantro/basil, chaat masala, salt, and pepper**.  
   - Taste and adjust seasoning (should be tangy, cheesy, with a hint of sweetness).  

2. **Assemble Samosas**  
   - Cut samosa wraps into **long triangles** (if using spring roll sheets, cut into strips).  
   - Place **1-2 tbsp filling** near the wide end, fold into a triangle, sealing edges with water.  
   - Optional: Sprinkle extra **parmesan on top** before frying/baking.  

3. **Cook**  
   - **Deep Fry:** Heat oil to 350°F (175°C), fry until golden (2-3 mins). Drain on paper towels.  
   - **Bake:** Brush with oil, bake at 375°F (190°C) for 15-20 mins until crisp.  

4. **Serve**  
   - Garnish with **more parmesan and chopped herbs**.  
   - Pair with **marinara sauce or mint chutney** for dipping!  

### **Why It Works**  
- **Garlic-herb pickle** = tangy, pizza-like depth.  
- **Parmesan** adds umami; **pineapple** balances with sweetness.  
- **Store-bought wraps** save time—crispy, flaky, and easy!  

### **Paratha Alternative**  
- Stuff the filling into **paratha dough**, roll, and pan-fry like a **stuffed flatbread**. Serve with a side of marinara!  

Let me know if you want adjustments—this combo is *next-level* fusion! 😋🔥

**Reply by shane0** [reply]
# Italian-Style Garlic & Herb Pickle 

(Fusion Achar) recipe using easy-to-find ingredients. It’s tangy, garlicky, and has pizza sauce vibes—perfect as a condiment for bread, pasta, or grilled meats!  

### **Ingredients (Small Batch)**  
- **1 cup peeled garlic cloves** (about 2 whole bulbs)  
- **½ cup sun-dried tomatoes in oil** (chopped, reserve some oil)  
- **2 tbsp red wine vinegar** (or apple cider vinegar)  
- **2 tbsp olive oil** (use oil from sun-dried tomatoes for extra flavor)  
- **1 tsp dried oregano**  
- **1 tsp dried basil**  
- **½ tsp red chili flakes** (adjust to taste)  
- **½ tsp black pepper**  
- **½ tsp salt** (adjust to taste)  
- **1 tsp sugar** (optional, balances acidity)  

### **Instructions**  
1. **Prep Garlic** – Blanch the garlic cloves in boiling water for **2 minutes** to mellow the sharpness (optional, but recommended). Drain and pat dry.  
2. **Mix Ingredients** – In a clean glass jar, combine garlic, sun-dried tomatoes, vinegar, olive oil, oregano, basil, chili flakes, black pepper, salt, and sugar.  
3. **Marinate** – Stir well, ensuring everything is coated in oil and vinegar. Add a little more oil if needed to submerge the garlic.  
4. **Rest & Ferment** – Seal the jar and let it sit at **room temperature for 1 day**, then refrigerate for **at least 3 days** before eating. The flavors will deepen over time.  

### **Serving Ideas**  
- Spread on **toast or bruschetta** (like a garlicky pizza sauce).  
- Toss with **pasta or salads** for extra flavor.  
- Pair with **cheese, grilled chicken, or sandwiches**.  

### **Storage**  
- Keeps for **2-3 weeks** in the fridge (if it lasts that long!).  

This fusion achar is **bold, herby, and addictive**—like a cross between Italian antipasto and Indian pickle! Let me know if you’d like any tweaks. 😊

**Reply by shane0** [reply]
#  **brine for achar (pickle)**

 is a saltwater solution that helps preserve and ferment the ingredients while enhancing flavor. Here’s an easy, versatile recipe:

### **Basic Achar Brine Recipe**  
*(Works for garlic, chilies, mixed veggies, or Italian-style pickle)*  

#### **Ingredients**  
- **1 cup water** (filtered or boiled & cooled)  
- **1 tbsp salt** (non-iodized: sea salt, kosher, or pickling salt)  
- **1 tbsp vinegar** (white, apple cider, or red wine vinegar – boosts acidity)  
- **1 tsp sugar** (optional, balances flavor)  

#### **Instructions**  
1. **Heat water** until warm (not boiling).  
2. **Dissolve salt (and sugar)** completely.  
3. **Cool to room temp**, then add vinegar.  

#### **How to Use**  
- Pour over **garlic, chilies, or veggies** in a clean jar (submerge fully).  
- For **fermented achar**: Leave at room temp 2–3 days (burp jar daily), then refrigerate.  
- For **quick achar**: Refrigerate immediately (ready in 24 hrs).  

#### **Variations**  
- **Spiced brine**: Add ½ tsp mustard seeds, ¼ tsp turmeric, or chili flakes.  
- **Sweet & tangy**: Increase sugar to 2 tsp + 2 tbsp vinegar.  

**Tip**: Always use sterilized jars, and ensure ingredients stay submerged to prevent mold.  

Let me know if you’re pickling something specific—I can tweak the brine! 😊

**Reply by shane0** [reply]
# spicy brine

To make your **achar (pickle) brine spicy**, you can easily amp up the heat with a few simple additions. Here’s how to turn up the spice level while keeping the brine balanced and flavorful:

---

### **Spicy Achar Brine Recipe**  
*(Ideal for garlic, chilies, mixed veggies, or fusion pickles)*  

#### **Ingredients**  
- **1 cup water** (filtered or boiled & cooled)  
- **1 tbsp salt** (non-iodized: pickling salt, kosher, or sea salt)  
- **1–2 tbsp vinegar** (white, apple cider, or red wine)  
- **1–3 tsp red chili flakes** (adjust to heat preference)  
- **1–2 fresh green or red chilies** (sliced, for extra kick)  
- **½ tsp black peppercorns** (lightly crushed)  
- **½ tsp mustard seeds** (optional, adds pungency)  
- **1–2 cloves garlic** (crushed, for depth)  

#### **Instructions**  
1. **Heat water** until warm (not boiling), then dissolve **salt**.  
2. **Add chili flakes, fresh chilies, peppercorns, mustard seeds, and garlic**.  
3. **Let steep for 5–10 minutes** (to infuse spice).  
4. **Cool completely**, then add **vinegar**.  
5. **Pour over vegetables/achar ingredients** in a sterilized jar, ensuring full submersion.  

#### **Spice Boosters (Choose Any)**  
- **1–2 dried red chilies** (whole or crushed)  
- **½ tsp cayenne powder** (for intense heat)  
- **1 tsp grated ginger** (adds warmth + spice)  
- **½ tsp Sichuan peppercorns** (for numbing heat)  

---

### **How to Adjust Heat Levels**  
- **Mild**: Use 1 tsp chili flakes + 1 fresh chili.  
- **Medium**: Use 1 tbsp chili flakes + 2 chilies + black pepper.  
- **Very Hot**: Add cayenne powder + extra chilies.  

### **Storage Tips**  
- **Fermented spicy achar**: Leave at room temp for 1–2 days (burp jar daily), then refrigerate.  
- **Quick spicy achar**: Refrigerate immediately (ready in 24 hours).  

This brine will give your achar a **fiery, complex heat** while preserving crunch and flavor. Let me know if you want even more extreme spice! 🔥🌶️

**Comment by shane0** [comment]
#  **Tepache** (pronounced *teh-PAH-chay*)

—a lightly fermented, tangy-sweet beverage made from **pineapple rinds, spices, and piloncillo (unrefined cane sugar)**. It’s a traditional probiotic drink with a fizzy, refreshing taste, often sold by street vendors in Mexico.  

---

### **Quick & Easy Tepache Recipe**  
*(Fermented in 2–3 days!)*  

#### **Ingredients**  
- **Rinds of 1 ripe pineapple** (organic preferred, with some flesh left on)  
- **1 cup piloncillo** (or brown sugar/panela)  
- **1 cinnamon stick**  
- **2–3 cloves** (optional)  
- **1 star anise** (optional, for depth)  
- **4 cups water** (filtered, non-chlorinated)  
- **1-inch piece ginger** (sliced, optional for extra zing)  

#### **Instructions**  
1. **Prep Pineapple**: Wash the pineapple well. Cut off the rind (save the fruit for eating).  
2. **Mix**: In a large jar, combine pineapple rinds, sugar, spices, and water. Stir to dissolve sugar.  
3. **Ferment**: Cover with a cloth or loose lid (to allow airflow). Let sit at **room temperature** for **2–3 days**, stirring once daily. Tiny bubbles will form—this means fermentation is working!  
4. **Strain & Bottle**: After 2 days, strain into bottles (leave 1–2 inches of headspace). Seal tightly and let sit **another 12–24 hours** to carbonate (like soda).  
5. **Enjoy!** Chill and serve over ice. It should taste **tangy, slightly sweet, and fizzy**.  

#### **Flavor Variations**  
- **Spicy**: Add 1–2 sliced jalapeños or a dash of cayenne.  
- **Fruity**: Toss in apple slices or mango peels during fermentation.  
- **Boozy**: Ferment longer (4–5 days) or mix with beer (*chela preparada*).  

---

### **Why It’s Unique**  
- **Probiotic**: Naturally fermented, good for gut health.  
- **Zero Waste**: Uses pineapple scraps (rinds + core).  
- **Refreshing**: Less sweet than soda, with a funky, tropical twist.  

Perfect for summer! Let me know if you want adjustments—it’s hard to mess up. 😊🍍

**Comment by shane0** [comment]
# Marco

**Reply by shane0** [reply]
Pasta
https://youtu.be/5lMiyNUlxAw

**Comment by shane0** [comment]
# carrots

The spicy, shredded carrot dish you're describing is likely a version of koreiskaya morkovka (Korean carrots), a popular salad in many post-Soviet countries, including Lithuania. While not originally Lithuanian, it has become a staple dish due to historical influences. It gets its signature tangy, spicy, and slightly fermented flavor from a hot oil dressing mixed with spices and vinegar. [1, 2, 3, 4, 5]  
Recipe for spicy shredded carrots 
Ingredients 

• Carrots: 1 lb, peeled and shredded into long, thin strips 
• Garlic: 3-4 cloves, finely minced or grated 
• Onion: ½ cup, thinly sliced 
• Vegetable oil: ¼ cup 
• White vinegar or rice wine vinegar: 2 tbsp 
• Salt: 1 tsp, or to taste 
• Sugar: ½ tsp 
• Spices: 1 tsp ground coriander, ½ tsp paprika, ¼ tsp cayenne pepper or red chili flakes (adjust for desired heat) [1, 3, 6, 7, 8]  

Instructions 

1. Prep the carrots: Shred the carrots using a box grater, julienne peeler, or the appropriate attachment on a food processor. Place the shredded carrots in a large bowl. 
2. Soften the carrots: Add the salt and sugar to the carrots. Mix well with your hands and let sit for 15 minutes. This will draw out some moisture and soften the carrots. 
3. Combine the vegetables: Add the minced garlic and sliced onions to the carrots. 
4. Heat the oil: In a small saucepan, heat the vegetable oil over medium-high heat until it just begins to smoke. Do not let it burn. 
5. Bloom the spices: Pour the hot oil directly over the spices (coriander, paprika, and cayenne) in a separate small bowl. Stir to combine. The heat from the oil will "bloom" the spices, intensifying their flavor. 
6. Mix the dressing: Immediately pour the spiced oil and the vinegar over the carrots, garlic, and onions. Toss everything together until the carrots are evenly coated. 
7. Marinate and serve: For the best flavor, cover the bowl and refrigerate for at least 30 minutes, allowing the carrots to marinate and the flavors to meld. The salad tastes even better the next day and can be refrigerated for up to 3 days. [1, 3, 9, 10, 11]  

AI responses may include mistakes.

[1] https://www.tiktok.com/discover/shredded-carrot-serrano-chili-salad-cook%E2%80%99s-kitchen[2] https://www.olgasflavorfactory.com/holiday-recipes/russian-carrot-salad-koreiskaya-morkovka/[3] https://thedeliciouslife.com/spicy-carrot-salad/[4] https://en.wikipedia.org/wiki/Morkovcha[5] https://somebodyfeedseb.com/korean-style-spicy-carrot-slaw/[6] https://www.facebook.com/EatingWell/videos/this-viral-chili-carrot-ribbon-salad-inspired-by-cassyeungmoney-is-the-tastiest-/1345314146570750/[7] https://www.tiktok.com/@thebalancedbowl/video/7526239721970797854[8] https://www.food.com/recipe/spicy-carrot-salad-138115[9] https://www.cooksnapbake.com/blog/spicycarrotribbonsalad[10] https://homemadeonaweeknight.com/2025/07/22/chili-crisp-carrot-salad/[11] https://sugarspiceandglitter.com/shredded-carrot-salad/
